operator/src/main/resources/scripts/startServer.sh

```diff
@@ -188,12 +206,109 @@ checkEnv \
   AS_SERVICE_NAME || exitOrLoop
 
 trace "LOG_HOME=${LOG_HOME}"
+trace "DATA_HOME=${DATA_HOME}"
 trace "SERVER_OUT_IN_POD_LOG=${SERVER_OUT_IN_POD_LOG}"
 trace "USER_MEM_ARGS=${USER_MEM_ARGS}"
 trace "JAVA_OPTIONS=${JAVA_OPTIONS}"
+trace "KEEP_DEFAULT_DATA_HOME=${KEEP_DEFAULT_DATA_HOME}"
+trace "LINK_SERVER_DATA_DIR=${LINK_SERVER_DATA_DIR}"
+
+#
+# If DATA_HOME env variable exists than this implies override directory (dataHome attribute of CRD) specified
+# so we need to try and link the server's 'data' directory to the centralized DATA_HOME directory
+# If 'KEEP_DEFAULT_DATA_HOME' env variable is defined then we will NOT link the server's 'data' directory to the
+# centralized DATA_HOME directory and instead keep the server's 'data' directory in its default location of
+# ${DOMAIN_HOME}/servers/${SERVER_NAME}/data
```

Comment:
Please add an explanation why we have this experimental code:   something like "This experimental code handles the specific case of services that don't provide a configurable way to control the location of their file data, specifically it's to cover stateful web-apps that use file state instead of database state.  The strongly recommended work-around is to have the web-app use database storage."  (I don't know web-app terminology so that might not be quite right.)

Reply:
On a related note, I updated your push description with a similar statement.  Please review the change and fix it up.